manifest.yaml

```diff
@@ -167,6 +167,18 @@ arguments:
     schema:
       type: number
     description: Overrides the high traffic threshold in the "http_success_rate_low" terraform module in monitoring/datadog.tf.
+  terraform.datadog.http.latency.count.highCount:
```

Comment:
Can you add a section to the docs for these? https://github.com/getoutreach/stencil-golang/blob/main/docs/arguments.md

Reply:
Does it exist for the other variables, such as `http_success_rate_high_traffic_percentile`?

Reply:
Ah, sorry, I should've noted. Just at the end is fine, it was missed in past PRs but just an entry _about_ the argument is good (like the other arguments)

Reply:
Added